### ISSUE TITLE
Sync button not working as expected

### ISSUE THREAD

**Issue by Nanush7** [thewca/wca-live#50]
First of all, sorry about my english.
Last competition, me and the delegate, spent a while trying to add a competitor to a category which he had not selected when registering (so we edited the register from the WCA page). The problem is that the sync button does not synchronize the categories, it took us a while to realize that in the page of the category there was an option to add competitors.

Basically, I think the synchronization button should also update the competitors per category.

**Comment by jonatanklosko** [comment]
To clarify, the current behavior is expected, once you open a round filling it with empty results, the results are only touched using WCA Live, so any changes on the WCA website don't affect the competitor list. As you mentioned there's a dedicated option to add a competitor to a round as well.
However this particular scenario of registering someone and synchronizing has lead to confusion several times and I agree that it makes sense to handle it automatically.

**Comment by Nanush7** [comment]
Yes, I agree with you. What I mean with "not working as expected" is that a person who has never used it may think it works in a different way, synchronizing the whole register.

I don't think it's too important, but it would be great if you can change this.

**Comment by jonatanklosko** [comment]
Fixed in 335c2235103eca025c1a32690d658841271c88f3.
